test/functional/data/rpc_decodescript.json

```diff
@@ -38,38 +36,21 @@
         "6a00",
         {
             "asm": "OP_RETURN 0",
-            "type": "nulldata",
-            "p2sh": "2NG8CqGyR16jkZU5H7J9WM5xpCT6Fpw6bww"
+            "type": "nulldata"
         }
     ],
     [
         "6aee",
         {
             "asm": "OP_RETURN OP_UNKNOWN",
```

Comment:
(not in this PR) it would be nice to have some test entries that do give `segwit` output

Reply:
good first issue ;)